sssom/cli.py

```diff
@@ -573,7 +575,7 @@ def merge(inputs: Tuple[str, str], output: TextIO, reconcile: bool = True):
     # If > 2 input files, iterate through each one
     # and merge them into the merged file above
     if len(inputs) > 2:
-        for input_file in inputs[2:]:
+        for input_file in inputs[2:]:  # type: str
```

Comment:
I don't like this, what is it supposed to refer to? let's strictly only use python syntax for type annotations, and never the comment syntax

Reply:
That's what I thought too. But it was complaining about the type and hence I made that comment to address it.

Reply:
Oh I see now. the type annotation is wrong for inputs. it should be `Sequence[str]`, not `Tuple[str, str]`

Reply:
I made the change and committing it. Thank you so much @cthoyt !

Reply:
Thanks :)